Documentation/Languages/Index.rst

```diff
@@ -40,7 +40,34 @@ Click on the "New record" icon and define your new language.
 
 
 Give your new language a title, relate it to an official language
-and choose a flag for it. Save and close, you're done.
+and choose a flag for it. Save and close.
+
+
+.. _languages-site-configuration:
+
+Making the language available
```

Comment:
I'm curious whether you consider that an editor or integrator task.
This manual is for editors, not integrators.

Would it make sense to move this new section to https://docs.typo3.org/m/typo3/guide-frontendlocalization/10.4/en-us/SettingUpLanguages/Index.html ?

We can add some hints, e.g. if something is missing. So in case an editor wants to translate, but that's not possible, he knows that it is possible and who he has to contact, e.g. his agency.

What do you think?

Reply:
Yes, i agree that the whole setup of new languages is an integrator task. So if the editor's tutorials should strictly contain editor's tasks i would remove the sections "Defining a new language" and "Making the language available" and move them to the integrator's corner.

In this tutorial i would add a hint as you suggested with redirecting to "your agency" or to the integrator's corner "[.. ] if your activities are not limited to editorial work" (as mentioned in page "Next Steps").